### DISCUSSION TITLE
First transaction was successful but isn't reflecting in the wallet

### DISCUSSION THREAD

**Discussion by phlenfyl** [Cyfrin/foundry-full-course-cu#2844] [General]
I don't know if this has to do with region issue. It didn't agree to link until I used a vpn, then it went, shows successful but isn't reflecting in my wallet.  I need help please. Thanks

**Comment by EngrPips** [comment]
What is not reflecting sir?

**Reply by phlenfyl** [reply]
after connecting my wallet address to the faucets chain line, I used the ethereum Sepolia to get token, it was successful after using vpn but couldn't see the token in my metamask wallet
![Screenshot 2024-11-10 185245](https://github.com/user-attachments/assets/4a38fb72-c11b-4a3b-a609-cf4c3a791234)
![Screenshot 2024-11-10 215249](https://github.com/user-attachments/assets/53997b17-1fd4-419f-b673-26d8c4b85b62)

**Reply by EngrPips** [reply]
Did you get `Link` or `ETH`?

**Reply by phlenfyl** [reply]
I got Link which is 250, when i clicked on it, it gave me this
![Screenshot 2024-11-11 121335](https://github.com/user-attachments/assets/51c65bb5-1780-4c32-9471-0a403e084665)

**Reply by LordAdhaar** [reply]
You need to add the link token to your wallet

Click on + Import tokens in the Tokens tab 
For LINK: Token Contract Address is 0x779877A7B0D9E8603169DdbD7836e478b4624789

**Reply by EngrPips** [reply]
Thanks for the heads-up @LordAdhaar

**Reply by LordAdhaar** [reply]
Just trying to give back to the community. Once I learn more I will be able to make better and more complex contributions to these questions. Also, this is the correct answer right? @EngrPips.

**Reply by EngrPips** [reply]
Yeah, your answer is correct. They need to add the token to their wallet.

**Reply by phlenfyl** [reply]
Thanks @LordAdhaar  I was able to add the token to my wallet. My SepoliaETH is still zero. I thought it was going to add the SepoliaETH @EngrPips 
![Screenshot 2024-11-11 192911](https://github.com/user-attachments/assets/831df040-af60-408d-a056-8048c2839a6f)

**Reply by LordAdhaar** [reply]
I don't know which faucet you selected. If you selected. There are two ethereum sepolia faucets one drops LINK and other ETH. Make sure you selected ETH. 

Alternatively you can get test Eth from 
https://cloud.google.com/application/web3/faucet/ethereum/sepolia

**Reply by EngrPips** [reply]
Nice heads-up

**Reply by phlenfyl** [reply]
Thanks @LordAdhaar . Actually when I selected ETH it failed saying I needed 1 LINK on Ethereum Mainnet which I couldn't get.
Thanks once again @EngrPips  @LordAdhaar . I can now proceed to the next

**Reply by EngrPips** [reply]
You can use this faucet https://sepolia-faucet.pk910.de/ to get some sepolia eth
